hbs-helpers.js

```diff
@@ -0,0 +1,26 @@
+"use strict";
+
+const HBSHelpers = {
+  init(hbs) {
+
+    hbs.registerHelper("prettyDate", function(date, options) {
```

Comment:
Me thinks `options` is unused.
This also feels a bit too locale specific. What about something like `toLocaleDateString()`?

```js
return new Date(date).toLocaleDateString('en-US', {year: 'numeric', month: 'long', day: 'numeric'});
```